tests/all-in-one.sh

```diff
@@ -2,6 +2,17 @@
 
 cd "$(mktemp -d ${TMPDIR:-/tmp}/dl-XXXXXXX)"
 
+# recipe from https://stackoverflow.com/a/4024263/1265472
+# sort -V seems to work on OSX yoh has access to.
+verlte() {
+    [  "$1" = "`echo -e "$1\n$2" | sort -V | head -n1`" ]
+}
+
+verlt() {
+    [ "$1" = "$2" ] && return 1 || verlte $1 $2
+}
```

Comment:
note: these might come handy later to make version dependent testing of rclone, e.g. as it would have been needed for 1.29 (disabled in github workflows matrix)